partner_delivery_zone/tests/test_partner_delivery_zone.py

```diff
@@ -0,0 +1,172 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from lxml import etree
+
+from odoo.tests import tagged
+
+from odoo.addons.base.tests.common import BaseCommon
+
+
+@tagged("post_install", "-at_install")
+class TestPartnerDeliveryZone(BaseCommon):
+    at_install = False
+    post_install = True
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.warehouse = cls.env.ref("stock.warehouse0")
+        cls.delivery_zone_a = cls.env["partner.delivery.zone"].create(
+            {"name": "Delivery Zone A", "code": "10"}
+        )
+        cls.delivery_zone_b = cls.env["partner.delivery.zone"].create(
+            {"name": "Delivery Zone A", "code": "10"}
+        )
+        cls.partner = cls.env["res.partner"].create(
+            {"name": "test", "delivery_zone_id": cls.delivery_zone_a.id}
+        )
+        cls.product = cls.env["product.product"].create({"name": "test"})
+        so = cls.env["sale.order"].new(
+            {
+                "partner_id": cls.partner.id,
+                "order_line": [
+                    (
+                        0,
+                        0,
+                        {
+                            "name": cls.product.name,
+                            "product_id": cls.product.id,
+                            "product_uom_qty": 10.0,
+                            "product_uom": cls.product.uom_id.id,
+                            "price_unit": 1000.00,
+                        },
+                    )
```

Comment:
Could you use the [Command](https://github.com/odoo/odoo/blob/370de95190cf07f17281f0ed6365b6806663cac2/odoo/fields.py#L4227-L4256) class? It is more readable.

```suggestion
                   Command.create(
                        {
                            "name": cls.product.name,
                            "product_id": cls.product.id,
                            "product_uom_qty": 10.0,
                            "product_uom": cls.product.uom_id.id,
                            "price_unit": 1000.00,
                        },
                    )
```

Reply:
done! didn't know about the Command i will start using it from now! thx!